plugins/cilium-cni/main.go

```diff
@@ -534,6 +539,8 @@ func cmdAdd(args *skel.CmdArgs) (err error) {
 		if err != nil {
 			return fmt.Errorf("unable to prepare IP addressing for '%s': %s", ep.Addressing.IPV6, err)
 		}
+		// Add to the result the Interface as index of Interfaces
+		ipConfig.Interface = cniTypesV1.Int(len(res.Interfaces))
```

Comment:
nit: this comment isn't particularly correct. Instead, can you put something more descriptive?

e.g. "set the addresses interface index to that of the container-side veth"?